patterns/featured-categories-grid.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Title: Featured Category Grid
+ * Slug: woocommerce-blocks/featured-category-grid
+ * Categories: WooCommerce
+ */
+
+$cols = 3;
+
+$categories = array_values(
+	get_categories(
+		array(
+			'taxonomy'   => 'product_cat',
+			'hide_empty' => true,
+		)
+	)
+);
+
+$rows = intdiv( count( $categories ), $cols ) + count( $categories ) % $cols;
```

Comment:
I believe this could be simplified to:
```suggestion
$rows = ceil( count( $categories ) / $cols );
```